.github/workflows/verify-generation-config.yaml

```diff
@@ -30,3 +72,40 @@ jobs:
           -e "REPO_BINDING_VOLUMES=${repo_volumes}" \
           gcr.io/cloud-devrel-public-resources/java-library-generation:latest \
           python /src/generate_repo.py generate --generation-config-yaml=/workspace/google-cloud-java/generation_config.yaml --repository-path=/workspace/google-cloud-java
+    - name: generate pull request description
+      id: generate-description
+      shell: bash
+      run: |
+        # the pr description (body) will be available in pr-description.txt after
+        # running this command.
+        set -x
+        docker run --rm \
+          ${repo_volumes} \
+          -v /tmp:/tmp \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          -e "RUNNING_IN_DOCKER=true" \
+          -e "REPO_BINDING_VOLUMES=${repo_volumes}" \
+          gcr.io/cloud-devrel-public-resources/java-library-generation:latest \
+          python /src/generate_pr_description.py generate --generation-config-yaml=/workspace/google-cloud-java/generation_config.yaml --baseline-commit="${baseline_commit}"
+      env:
+        baseline_commit: ${{ env.baseline_commit }}
+    - name: create or update the pull request
+      shell: bash
+      run: |
+        title="chore: generate libraries at $(date)"
+        git add java-* pom.xml gapic-libraries-bom/pom.xml versions.txt generation_config.yaml
+        git commit -m "${title}"
+        if [ -z "${pr_num}" ]; then
+          git remote add monorepo https://cloud-java-bot:${GH_TOKEN}@github.com/${{ github.repository }}.git
+          git fetch -q --unshallow monorepo
+          git push -f monorepo "${branch_name}"
+          set -x
+          gh pr create --title "${title}" --label "owlbot:run" --head "${branch_name}" --body "$(cat pr_description.txt)"
```

Comment:
Why do we still need `owlbot:run` here?

Reply:
The check is not removed yet.

Reply:
I think we should remove the label here and disable the check for this branch, otherwise owlbot postprocessing would still be run against this PR right?

Reply:
+1 to removing the required check for owlbot postprocessor and the label

Reply:
Done.